packages/farms/constants/v3/index.ts

```diff
@@ -40,7 +40,15 @@ export const bCakeFarmBoosterV3Address: Addresses = {
   [ChainId.BSC]: '0x695170faE243147b3bEB4C43AA8DE5DcD9202752',
   [ChainId.BSC_TESTNET]: '0x56666300A1E25624489b661f3C6c456c159a109a',
 }
-export const bCakeFarmBoosterVeCakeAddress: Addresses = {
+export const bCakeFarmBoosterV3VeCakeAddress: Addresses = {
```

Comment:
is this farm booster address or vecake address?

Reply:
it's different version og farm booster address